homeassistant/helpers/device_registry.py

```diff
@@ -33,6 +33,16 @@
 CONNECTION_ZIGBEE = "zigbee"
 
 
+@attr.s(slots=True, frozen=True)
+class DeletedDeviceEntry:
+    """Deleted Device Registry Entry."""
+
+    config_entries: Set[str] = attr.ib()
```

Comment:
Let's make this a list if we convert `connections` and `identifiers` too ?

Reply:
There are some set operations performed on the `config_entries` member though, e.g. here:
`if config_entries == {remove_config_entry_id}:`